src/Raven.Server/Documents/Indexes/Persistence/Corax/JintCoraxDocumentConverter.cs

```diff
@@ -359,37 +367,49 @@ private static JsValue TryDetectDynamicFieldCreation(string property, ObjectInst
                 var propertyNameAsString = propertyName.AsString();
                 if (string.Equals(propertyNameAsString, nameof(CreateFieldOptions.Indexing), StringComparison.OrdinalIgnoreCase))
                 {
-                    field.Indexing = GetEnum<FieldIndexing>(optionValue, propertyNameAsString);
+                    fieldIndexing = GetEnum<FieldIndexing>(optionValue, propertyNameAsString);
 
                     continue;
                 }
 
                 if (string.Equals(propertyNameAsString, nameof(CreateFieldOptions.Storage), StringComparison.OrdinalIgnoreCase))
                 {
                     if (optionValue.IsBoolean())
-                        field.Storage = optionValue.AsBoolean()
+                        fieldStorage = optionValue.AsBoolean()
                             ? FieldStorage.Yes
                             : FieldStorage.No;
                     else
-                        field.Storage = GetEnum<FieldStorage>(optionValue, propertyNameAsString);
+                        fieldStorage = GetEnum<FieldStorage>(optionValue, propertyNameAsString);
 
                     continue;
                 }
 
                 if (string.Equals(propertyNameAsString, nameof(CreateFieldOptions.TermVector), StringComparison.OrdinalIgnoreCase))
                 {
-                    field.TermVector = GetEnum<FieldTermVector>(optionValue, propertyNameAsString);
+                    //ignore
```

Comment:
Can you elaborate on that?

Reply:
This is Corax's JINT converter, TermVector is not used anyway so we can ignore it.

Reply:
Maybe a better comment would be nice here